docs/src/starknet/account.md

```diff
@@ -48,7 +48,7 @@ Do the following to start interacting with the Starknet:
       --url http://127.0.0.1:5050 \
       account deploy
       --name some-name \
-      --max-fee 864600000000
+      --max-fee 9999999999999
```

Comment:
Perhaps we could also adjust the output above to match these numbers? Apart from that I think the example below with salt should be updated as well.

Reply:
Thought about it, but these are outputs (not inputs), so I'm rather reluctant. Users will see something different either way

Reply:
That's fine. Regarding the second part I meant updating `--max-fee` under [Custom Account Contract](https://foundry-rs.github.io/starknet-foundry/starknet/account.html#custom-account-contract) example.

Reply:
in this case, i think it may make sense to keep it, as the input is directly taken from the output of a previous command; I think it may be confusing if we discredit the output of the first command completely